CTW/Cake_CTW/map.json

```diff
@@ -0,0 +1,109 @@
+{
+	"name": "Cake",
+	"authors": ["Clijmart, Sam, mepii"],
```

Comment:
You might want to organize the authors as ["Clijmart", "Sam". "mepii"]. And try to make sure you are using the builder's full IGN. I'm not really too sure who "Sam" is.